setup.py

```diff
@@ -39,6 +39,7 @@
         'cartopy>=0.18.0',
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
+#        'seaborn==0.12',
```

Comment:
I think PEP8 is complaining about this, perhaps the comment needs to be in front of the quote?